STYLE.md

```diff
@@ -52,16 +59,26 @@ following abbreviations we assume readers will already know.
 - UTXO (Unspent Transaction Output)
 - vbyte (virtual byte)
 
+Introduce sighash with "signature hash (sighash)", but don't introduce sighash
+types (`SIGHASH_SINGLE`, `SIGHASH_ALL`, etc) since they're technical terms
+and not abbreviations.
+
 ### Compound words
 
 | Use | Don't use |
 |-|-|
 | block chain | blockchain |
+| coinjoin | Coinjoin or coinJoin or coin-join |
 | mainnet | main net |
 | multisig | multi-sig |
+| multiparty | multi-party |
+| MuSig | Musig or muSig or musig |
 | offchain | off-chain |
 | onchain | on-chain |
+| preimage | pre-image |
 | redeemScript | redeem script |
+| single-sig | singlesig |
+| soft fork/hard fork | softfork/hardfork |
```

Comment:
> softfork/hardfork

I feel like have seen the hyphen versions out there as well. Consider adding if you agree:

> soft-fork/hard-fork

Reply:
I think hyphens should only be used for compound adjectives, e.g. "Foo proposed a soft-fork change."